servers/plugins/org.fusesource.ide.server.fuse.core/OSGI-INF/l10n/bundle.properties

```diff
@@ -8,30 +8,30 @@
 # Contributors:
 #     JBoss by Red Hat - Initial implementation.
 ##############################################################################
-runtimeTypeFuseESBLabel_60=Red Hat JBoss Fuse 6.0
```

Comment:
6.x ones need to be kept with Red Hat Jboss Fuse, see https://issues.jboss.org/browse/FUSETOOLS-2843?focusedCommentId=13545002&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-13545002